src/concepts/workflows/flow.md

```diff
@@ -0,0 +1,84 @@
+# Workflow Execution and Data Flow
+
+The order of Workflow operation can be split into two concepts:
+
+## Workflow Execution
+
+A Workflow processes Nodes in sequential order:
+
+1. Beginning at the root of the Node tree with either a "Start" or "Intercept" Node.
+2. Every subsequent node in the chain is processed in order. The execution may take the path of another branch based on conditionals.
+3. The Workflow ends either when it reaches an explicit "End" Node or when no more Nodes are available in the chain.
+
+<img alt="Register command SDK." src="/_images/execution_flow.png" center/>
+
+## Workflow Data Flow
+
+Each Node in a Workflow has an input and output data type:
+
+::: info
+The `Choice` and `Code` types are just variations of `String`.
+:::
+
+``` rust
+pub enum NodeDataProp {
+    String {
+        alias: Alias,
+        name: String,
+        optional: bool,
+        default: Option<String>,
+    },
+    Bytes {
+        alias: Alias,
+        name: String,
+        optional: bool,
+    },
+    Choice {
+        alias: Alias,
+        name: String,
+        optional: bool,
+        values: Vec<String>,
+        default: Option<String>,
+    },
+    Code {
+        alias: Alias,
+        name: String,
+        description: Option<String>,
+        optional: bool,
+        default: Option<String>,
+        language: Option<String>,
+    },
+    Bool {
+        alias: Alias,
+        name: String,
+        default: bool,
+    },
+    Integer {
+        alias: Alias,
+        name: String,
+        min: i64,
+        max: i64,
+        default: Option<i64>,
+    },
+    Request {
+        alias: Alias,
+        name: String,
+        optional: bool,
+        saved: bool,
+    },
+    Response {
+        alias: Alias,
+        name: String,
+        optional: bool,
+        saved: bool,
+    },
+}
+```
+
+With this typed data system, even though Nodes are processed sequentially, you do not need a direct line between two Nodes in order to pass data from one to another.
```

Comment:
We use the alias system for that. Unsure if we explained it elsewhere, but this is the place to write about it.
So `[node_alias].[output_alias]` is the format we use.